src/2.0/types.ts

```diff
@@ -1,20 +1,31 @@
 // types generated by quicktype during postinstall phase
 import { OpenAttestationDocument as OpenAttestationDocumentV2 } from "../__generated__/schema.2.0";
-import { SchemaId, ObfuscationMetadata, ProofPurpose, ProofType } from "../shared/@types/document";
-
-export interface Proof {
-  type: ProofType;
-  created: string;
-  proofPurpose: ProofPurpose;
-  verificationMethod: string;
-  signature: string;
-}
-export interface Signature {
-  type: "SHA3MerkleProof";
-  targetHash: string;
-  proof: string[];
-  merkleRoot: string;
-}
+import { OpenAttestationHexString, ProofPurpose, ProofType, SchemaId } from "../shared/@types/document";
+import { Array as RunTypesArray, Literal, Partial, Record as RunTypesRecord, Static, String } from "runtypes";
+
+export const ObfuscationMetadata = Partial({
```

Comment:
![image](https://user-images.githubusercontent.com/27843274/110908125-54978700-8349-11eb-9df9-b3bcd75bb599.png)

is this what you expect?

Reply:
not exactly, but it's also not wrong as obfuscatedData maybe absent. It should work correctly. wdyt ?

Reply:
hmm idk, do you also require it to fail if it has wrong keys?